br/pkg/streamhelper/advancer_cliext.go

```diff
@@ -23,6 +24,13 @@ import (
 	"go.uber.org/zap"
 )
 
+var (
+	reAccessKey       = regexp.MustCompile(`access_key:\"[^\"]*\"`)
+	reSecretAccessKey = regexp.MustCompile(`secret_access_key:\"[^\"]*\"`)
+	reSharedKey       = regexp.MustCompile(`shared_key:\"[^\"]*\"`)
```

Comment:
Azure blob storage has another credential -- `account-name` and `sas-token`. The `sas-token` is also need to be redacted.